www/js/pages/trade/components/payout/payout.controller.js

```diff
@@ -48,7 +48,7 @@
                 if (/^\.\d+$/.test(vm.amount)){
                     vm.amount = `0${vm.amount}`;
                 }
-                vm.proposal.amount = vm.amount;
+                vm.proposal.amount = typeof vm.amount === "string" ? Number(vm.amount) : vm.amount;
```

Comment:
@nazaninreihani also you can use simply a `+` before `vm.amount` to change it to a number.
```suggestion
                vm.proposal.amount = +vm.amount;
```